src/containers/JobsTrainingPage/index.jsx

```diff
@@ -1,11 +1,14 @@
 import React from "react";
 import { Container } from "react-bootstrap";
 import JobsTrainingHeader from "../../components/JobsTrainingHeader";
+import JobsTrainingContent from "../../components/JobsTrainingContent";
 
 const JobsTrainingPage = () => {
+  // ToDo(hibaMachfej): Add search functionality to jobs/training page.
```

Comment:
did you just add this in this PR?

Reply:
No it's from the old PR, I did not have search functionality.